inst/pages/integrated_learner.qmd

```diff
@@ -19,161 +17,203 @@ advanced method, called cooperative learning [@Ding2022], which is also known as
 predictions from different data views to align through an agreement parameter
 ($\rho$).
 
-## Multi-omics Prediction and Classification of Binary IBD Disease Status
+## Prediction of fat and prebiotics in diet of rats
 
-In this chapter, we showcase examples of various integration paradigms (early,
-late, and intermediate fusion) using the R packages `multiview` [@Ding2022]
-and `SuperLearner` [@Van2007]. We make use of the publicly available source code
-from the multi-omics integrated framework (`IntegratedLearner`) proposed by
-@Mallick2024.
+As an example we would build a random forest classifier based on theHintikkaxOData 
+multi-omics profiles to classify fat and prebiotics in diet of rats. 
 
-## Input data
+### With validation set
 
-We use the publicly available Inflammatory Bowel Diseases (IBD) data from the 
-`curatedMetagenomicData` package [@Lloyd-Price2019], where we aim to 
-predict IBD disease status based on both taxonomic (species abundances) and 
-functional (pathway abundances) profiles.
+```{r valid-set}
 
-```{r load-pkg-data}
+###########################
+# load required libraries #
+###########################
+library(mia)
+library(SuperLearner)
+library(IntegratedLearner)
+library(tidyverse)
+library(cowplot)
 
-##################
-# Load iHMP data #
-##################
+########################
+# load multi-omic data #
+########################
+data("HintikkaXOData")
+mae <- HintikkaXOData
 
-library(curatedMetagenomicData)
-library(dplyr)
-library(tidyverse)
+######################################
+# Add validation set info to colData #
+######################################
 
-se_relative <- sampleMetadata |>
-  filter(study_name == "HMP_2019_ibdmdb") |>
-  returnSamples("relative_abundance", rownames = "short")
+colData(mae)[["valid"]] <- sample(c(TRUE, FALSE), nrow(colData(mae)), 
+                                  replace = TRUE)
 
-se_pathway <- sampleMetadata |>
-  filter(study_name == "HMP_2019_ibdmdb") |>
-  returnSamples("pathway_abundance", rownames = "short")
```

Comment:
It might be still good to show this. You can disable this code chunk so that it is not evaluated and then load the already loaded data so that for reader it seems that the data was fetched from databse

Reply:
I have removed the example.